lib/command/encrypt.js

```diff
@@ -78,17 +78,21 @@
     };
```

Comment:
Looks like the built version of this file is out of date. The doc string here on line 71 at least still is old.